ui-v2/app/templates/components/healthcheck-output.hbs

```diff
@@ -1,28 +1,8 @@
+{{! TODO: this component and its parent should be moved to a single component }}
```

Comment:
Is this TODO and the one above it something you want to get done now?

Reply:
Nope, in fact from not so long ago I've been trying to follow @backspace 's lead and use `TODO` for things to do at some point in the future in another PR and then use `FIXME` for things that I'd planned to do in the PR but I'm looking for feedback before I do it (or change to a `TODO`)